src/woo_publications/metadata/models.py

```diff
@@ -6,20 +6,26 @@
 from ordered_model.models import OrderedModel
 from treebeard.mp_tree import MP_Node
 
-from .constants import InformationCategoryOrigins
+from .constants import Origins
 from .manager import InformationCategoryManager, ThemeManager
 
 CUSTOM_CATEGORY_IDENTIFIER_URL_PREFIX = (
     "https://generiek-publicatieplatform.woo/informatiecategorie/"
 )
 
+CUSTOM_ORGANIZATION_URL_PREFIX = "https://generiek-publicatieplatform.woo/organization/"
```

Comment:
```suggestion
CUSTOM_ORGANISATION_URL_PREFIX = "https://generiek-publicatieplatform.woo/organisation/"
```
Let's stick to consistent British English spelling. The model name is correct.